src/Symfony/Component/Translation/Translator.php

```diff
@@ -419,8 +419,18 @@ private function getFallbackContent(MessageCatalogue $catalogue)
 
     private function getCatalogueCachePath($locale)
     {
+        $resources = array();
+        if (isset($this->resources[$locale])) {
+            foreach ($this->resources[$locale] as $resource) {
+                if (is_object($resource[1]) && !$resource[1] instanceof \Serializable) {
+                    $resource[1] = get_class($resource[1]);
```

Comment:
Me too, `spl_object_hash($resource[1])` doesn't solve the issue because when object is destroyed, its hash may be changed.
I think it's better to removing that part until we found the right way ??